js/samples/node/config.js

```diff
@@ -0,0 +1,8 @@
+(function() {
+  'use strict';
+
+module.exports.cfg = {
```

Comment:
weird alignment

Reply:
Also, can we stick to the comment style we use everywhere else? Which is:
```  // Log stash.``` (two spaces before double slash, and properly formatted sentences)

Reply:
Multiple projects, multiple requirements, sometimes I get crazy mixes :) I'll run through the comments to adjust styling, but I didn't catch about weird aligment. You meant we should keep two spaces offsets for everything inside an anonymous function? or get rid of spaces before 'use strict'? or either of them?